packages/webapp/src/components/Finances/DateRangeSelector/index.jsx

```diff
@@ -72,19 +70,6 @@ const FinanceDateRangeSelector = ({ hideTooltip }) => {
 
   return (
     <div className={styles.rangeContainer}>
-      <div className={styles.titleContainer}>
```

Comment:
I removed this bit because in the new designs there's no title for the date range selector. @SayakaOno is that okay?

Reply:
I think so! Thank you :)